pkgs/os-specific/linux/ch9344/default.nix

```diff
@@ -34,7 +35,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
```

Comment:
Please set license = gpl2Plus per https://github.com/WCHSoftGroup/ch9344ser_linux/blob/main/driver/ch9344.c#L10-L13

Reply:
Header file appears to be [GPL v2 only](https://github.com/WCHSoftGroup/ch9344ser_linux/blob/main/driver/ch9344.h#L1). So the whole package should be the lowest common denominator no?

Reply:
I think `GPL2 or later` is a mistake on their side. Since it derives from kernel code this doesn't make sense and it's effectively GPLv2-only anyhow...